BusyList/Handlers/DoneHandler.cs

```diff
@@ -0,0 +1,26 @@
+﻿using BusyList.Commands;
+using System;
+
+namespace BusyList.Handlers
+{
+    public class DoneHandler : IHandler<DoneCommand>
+    {
+        private readonly ITaskRepository _TaskRepository;
```

Comment:
Should be a lowercase t for this field (private fields follow the `_camelCase` rule).